libraries/ESP8266WiFi/src/include/DataSource.h

```diff
@@ -1,154 +0,0 @@
-/* DataSource.h - a read-only object similar to Stream, but with less methods
```

Comment:
It appeared afterwards that `Stream` integrates missing functionalities that allows itself to be a replacement for `DataSource`.